docs/usage/index.md

```diff
@@ -76,7 +76,33 @@ Renovate works on these platforms:
 
 Renovate is used by:
 
-![Renovate Matrix](./assets/images/matrix.png){ loading=lazy }
+![Logos of companies and projects that use Renovate](./assets/images/matrix.png){ loading=lazy }
+
+<details>
+<summary>List of companies and projects that use Renovate</summary>
+
+- Prisma
+- Netlify
+- Envoy
+- Condé Nast
+- Microsoft
+- Atlassian
+- Sourcegraph
+- Mozilla
+- Deloitte
+- Telus
+- Yarn
+- HashiCorp
+- Automattic
+- Algolia
+- eBay
+- Cypress
+- Red Hat
+- Financial Times
+- Uber
+- Buildkite
+
+</details>
```

Comment:
I guess that the current `<details>` and `<summary>` blocks will work on the published docs.

Alternatively, we can try a _collapsible_ admonition, which defaults to the collapsed state. Read the [Material for MkDocs, collapsible admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#collapsible-blocks) manual to learn more about that.